### PR TITLE
Release 0.78

### LINKED ISSUE #1764: Release the operator v0.78.0

### PR DESCRIPTION
**Pull request by jaronoff97**
closes #1764

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -571,6 +571,7 @@ The OpenTelemetry Operator *might* work on versions outside of the given range,
 
 | OpenTelemetry Operator | Kubernetes           | Cert-Manager        |
 |------------------------|----------------------|---------------------|
+| v0.78.0                | v1.19 to v1.27       | v1                  |
 | v0.77.0                | v1.19 to v1.26       | v1                  |
 | v0.76.1                | v1.19 to v1.26       | v1                  |
 | v0.75.0                | v1.19 to v1.26       | v1                  |
```
Comment (yuriolisa) [inline README.md:577]: Please remove the last one.

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,24 @@ Changes by Version
...
+- `operator`: Propagate Metadata.Annotations to PodSpec.Annotations (#900)
+- `operator`: Improve config validation for prometheus receiver and target allocator (#1581)
+
+### 🧰 Bug fixes 🧰
+
+- `operator`: fixes a previously undocumented behavior that a collector could not override the collector's app name (#1777)
+- `operator`: Fix issue where the operator's released image did not correctly set the default go auto-instrumentation version (#1757)
+- `pkg/collector, pkg/targetallocator`: fix issues related to prometheus relabel configs when target allocator is enabled (#958, #1622, #1623)
+
 ## 0.77.0
```
Comment (yuriolisa) [inline CHANGELOG.md:23]: Should the components links be present here?

**Review by yuriolisa — APPROVED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,36 @@ Changes by Version
...
+- `pkg/collector, pkg/targetallocator`: fix issues related to prometheus relabel configs when target allocator is enabled (#958, #1622, #1623)
+
+### Components
+
+* [OpenTelemetry Collector - v0.78.0](https://github.com/open-telemetry/opentelemetry-collector/releases/tag/v0.78.0)
+* [OpenTelemetry Contrib - v0.78.0](https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/tag/v0.78.0)
+* [Java auto-instrumentation - 1.25.0](https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases/tag/v1.25.0)
+* [.NET auto-instrumentation - 0.7.0](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/releases/tag/v0.7.0)
+* [Node.JS - 0.37.0](https://github.com/open-telemetry/opentelemetry-js-contrib/releases/tag/auto-instrumentations-node-v0.38.0)
+* [Python - 0.38b0](https://github.com/open-telemetry/opentelemetry-python-contrib/releases/tag/v0.38b0)
```
Comment (TylerHelmuth) [inline CHANGELOG.md]:
```suggestion
* [Node.JS - 0.39.1](https://github.com/open-telemetry/opentelemetry-js-contrib/releases/tag/auto-instrumentations-node-v0.39.1)
* [Python - 0.39b0](https://github.com/open-telemetry/opentelemetry-python-contrib/releases/tag/v0.39b0)
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,37 @@ Changes by Version
...
+
+- `operator`: fixes a previously undocumented behavior that a collector could not override the collector's app name (#1777)
+- `operator`: Fix issue where the operator's released image did not correctly set the default go auto-instrumentation version (#1757)
+- `pkg/collector, pkg/targetallocator`: fix issues related to prometheus relabel configs when target allocator is enabled (#958, #1622, #1623)
+
+### Components
+
+* [OpenTelemetry Collector - v0.78.0](https://github.com/open-telemetry/opentelemetry-collector/releases/tag/v0.78.0)
+* [OpenTelemetry Contrib - v0.78.0](https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/tag/v0.78.0)
+* [Java auto-instrumentation - 1.25.0](https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases/tag/v1.25.0)
```
Comment (pavolloffay) [inline CHANGELOG.md:29]: Java agent 1.26.0  is used

File: CHANGELOG.md [hunk 5]
```diff
@@ -21,8 +51,8 @@ Changes by Version
 * [OpenTelemetry Contrib - v0.77.0](https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/tag/v0.77.0)
 * [Java auto-instrumentation - 1.25.0](https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases/tag/v1.25.0)
 * [.NET auto-instrumentation - 0.7.0](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/releases/tag/v0.7.0)
-* [Node.JS - 0.37.0](https://github.com/open-telemetry/opentelemetry-js-contrib/releases/tag/auto-instrumentations-node-v0.38.0)
-* [Python - 0.38b0](https://github.com/open-telemetry/opentelemetry-python-contrib/releases/tag/v0.38b0)
+* [Node.JS - 0.39.1](https://github.com/open-telemetry/opentelemetry-js-contrib/releases/tag/auto-instrumentations-node-v0.39.1)
+* [Python - 0.39b0](https://github.com/open-telemetry/opentelemetry-python-contrib/releases/tag/v0.39b0)
```
Comment (TylerHelmuth) [inline CHANGELOG.md]:
```suggestion
* [Node.JS - 0.38.0](https://github.com/open-telemetry/opentelemetry-js-contrib/releases/tag/auto-instrumentations-node-v0.38.0)
* [Python - 0.38b0](https://github.com/open-telemetry/opentelemetry-python-contrib/releases/tag/v0.38b0)
```

**Review by pavolloffay — COMMENTED** [review]
LGTM, just one nit

**Pull request merged**